PowerEditor/src/MISC/PluginsManager/Notepad_plus_msgs.h

```diff
@@ -36,8 +36,10 @@ enum LangType {L_TEXT, L_PHP , L_C, L_CPP, L_CS, L_OBJC, L_JAVA, L_RC,\
 			   // Don't use L_JS, use L_JAVASCRIPT instead
 			   // The end of enumated language type, so it should be always at the end
 			   L_EXTERNAL};
+enum class LangAutoIndentType { Standard = 1, Extended, Custom };
+enum class MacroStatus { Idle, RecordInProgress, RecordStop, PlayingBack };

```

Comment:
Choose between `enum class MacroStatus { Idle, RecordInProgress, RecordStop, PlayingBack };` and
```
#define MACRO_IS_IDLE 0
#define MACRO_RECORDING_IN_PROGRESS 1
#define MACRO_RECORDING_HAS_STOPPED 2
#define MACRO_IS_PLAYINGBACK 3
```
The coexistence will make confusion.

Reply:
Left the enum class active, since the other `#defines` where just internal definitions. Left the code commented out, but if you want me to remove the comments also, just tell.